persistence/sql/persister_grant_jwk.go

```diff
@@ -90,7 +90,7 @@ func (p *Persister) GetPublicKey(ctx context.Context, issuer string, subject str
 	var data trust.SQLData
 	query := p.Connection(ctx).
 		Where("issuer = ?", issuer).
-		Where("subject = ?", subject).
+		Where("subject = ? OR allow_any_subject IS TRUE", subject).
```

Comment:
Actually, could you please add a test for the persister that checks whether this works? :)

Reply:
Of course!